stable/aws-for-fluent-bit/README.md

```diff
@@ -96,15 +116,80 @@ helm delete aws-for-fluent-bit --namespace kube-system
+| `s3.s3KeyFormatTagDelimiters` | A series of characters which will be used to split the tag into 'parts' for use with the s3_key_format option. See the in depth examples and tutorial in the documentation. |
+| `s3.staticFilePath` | Disables behavior where UUID string is automatically appended to end of S3 key name when $UUID is not provided in s3_key_format. $UUID, time formatters, $TAG, and other dynamic key formatters all work as expected while this feature is set to true. | `false`
+| `s3.usePutObject` | Use the S3 PutObject API, instead of the multipart upload API. When this option is on, key extension is only available when $UUID is specified in s3_key_format. | `false`
+| `s3.roleArn` | ARN of an IAM role to assume (ex. for cross account access). |
+| `s3.endpoint` | Custom endpoint for the S3 API. An endpoint can contain scheme and port. |
+| `s3.stsEndpoint` | Custom endpoint for the STS API. |
+| `s3.cannedAcl` | Predefined Canned ACL policy for S3 objects. |
+| `s3.compression` | Compression type for S3 objects. 'gzip' is currently the only supported value by default. If Apache Arrow support was enabled at compile time, you can also use 'arrow'. |
+| `s3.contentType` | A standard MIME type for the S3 object; this will be set as the Content-Type HTTP header. |
+| `s3.sendContentMd5` | Send the Content-MD5 header with PutObject and UploadPart requests, as is required when Object Lock is enabled. | `false`
+| `s3.autoRetryRequests` | Immediately retry failed requests to AWS services once. This option does not affect the normal Fluent Bit retry mechanism with backoff. Instead, it enables an immediate retry with no delay for networking errors, which may help improve throughput when there are transient/random networking issues. | `true`
+| `s3.logKey` | By default, the whole log record will be sent to S3. If you specify a key name with this option, then only the value of that key will be sent to S3. For example, if you are using Docker, you can specify log_key log and only the log message will be sent to S3. |
+| `s3.preserveDataOrdering` | Normally, when an upload request fails, there is a high chance for the last received chunk to be swapped with a later chunk, resulting in data shuffling. This feature prevents this shuffling by using a queue logic for uploads. | `true`
+| `s3.storageClass` | Specify the storage class for S3 objects. If this option is not specified, objects will be stored with the default 'STANDARD' storage class. |`
+| `s3.retryLimit`| Integer value to set the maximum number of retries allowed. Note: this configuration is released since version 1.9.10 and 2.0.1. For previous version, the number of retries is 5 and is not configurable. | 1
+|`s3.externalId`| Specify an external ID for the STS API, can be used with the role_arn parameter if your role requires an external ID.
+|`s3.extraOutputs`| Append extra outputs with value |`""`|
```

Comment:
same comment as on cloudwatch, make the language  really clear

Reply:
Done